.github/workflows/redeploy/redeploy_specific.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+
+iplist=$1
+
+for ip_address in $iplist
+do
+    echo $ip_address
+    ssh-keyscan -H $ip_address >> ~/.ssh/known_hosts
+    sleep 2
+
+    if [ "$ip_address" == "167.172.70.98" ] || [ "$ip_address" == "cloudretro.io" ] 
+    then
+        launchcommand="coordinator > /tmp/startup.log"
+    else
+        launchcommand="Xvfb :99 & worker --coordinatorhost cloudretro.io --zone \$zone > /tmp/startup.log"
```

Comment:
xvfb is used only for n64 OpenGL, so you might remove it in case if there are no n64 games on your hosts.